File: pset6/cash.py
from cs50 import get_float

while True:
    try:
        change = get_float("How much change is owed?")
        if change > 0:
            break
    except:
        print("", end="")
            
coins = 0
changeR = change * 100

while changeR >= 25:
    changeR = changeR - 25
    coins = coins + 1
while changeR >= 10:
    changeR = changeR - 10
    coins = coins + 1
while changeR >= 5:
    changeR = changeR - 5
    coins = coins + 1
while changeR >= 1:
    changeR = changeR - 1
    coins = coins + 1
    
print(coins)

File: pset1/Cash pset1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    
    
    
    float change;
     
    
    do 
    {
        
        //User Input
        change = get_float("How much change is owed?\n");
        
    }
    
    while (change < 0);
    
    
    //Round up change
    int changeR = round(change * 100);
    int coins = 0;    
    //Calculate coins  
    while (changeR >= 25)
    {
        changeR -= 25;
        coins ++;
    }
    
    while (changeR >= 10)
    {
        changeR -= 10;
        coins ++;
    }
    
    while (changeR >= 5)
    {
        changeR -= 5;
        coins ++;
    }
    
    while (changeR >= 1)
    {
        changeR -= 1;
        coins ++;
    }
    
    //Display Coins 
    printf("Coins Owed: ""%i\n", coins); 
    
    
}